plugins/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/util/ProjectFactory.java

```diff
@@ -142,25 +156,34 @@ public IProject createProject(IProgressMonitor monitor, Shell shell) {
 			project.setDefaultCharset(defaultCharset, subMonitor.newChild(1));
 			createFolders(project, subMonitor, shell);
 
-			// first run contributors...
-			if (contributors != null) {
-				IFileCreator fileCreator = new IFileCreator() {
+			IFileCreator fileCreator = new IFileCreator() {
+				@Override
+				public IFile writeToFile(CharSequence chars, String fileName) {
+					return ProjectFactory.this.writeToFile(chars, fileName, project, subMonitor);
+				}
+			};
 
-					@Override
-					public IFile writeToFile(CharSequence chars, String fileName) {
-						return ProjectFactory.this.writeToFile(chars, fileName, project, subMonitor);
-					}
-				};
-				for (IProjectFactoryContributor contributor : contributors) {
+			// first run early contributors...
```

Comment:
Are comments in this method necessary? References to bugs can be found in the history.